test/WebJobs.Extensions.Tests/Extensions/Http/HttpTriggerEndToEndTests.cs

```diff
@@ -104,6 +105,15 @@ public void BindToRequestAndPocoAtSameTime()
             Assert.Equal(request.Properties["result"], "abc"); // Verify resposne was set
         }
 
+        [Fact]
+        public void ClaimsPrincipalBinding()
+        {
+            var request = new HttpRequestMessage(HttpMethod.Get, "http://functions.com/api/TestIdentityBindings");
+
+            var method = typeof(TestFunctions).GetMethod("TestIdentityBindings");
```

Comment:
Instead of hardcoding "TestIdentityBindings", it's more robust to do `nameof(TestIdentityBindings)`.